src/isomorphic/classic/types/__tests__/ReactPropTypes-test.js

```diff
@@ -45,6 +45,13 @@ function typeCheckPass(declaration, value) {
   expect(error).toBe(null);
 }
 
+function typeCheckWarn(propTypeFunc, message) { 
```

Comment:
This helper is a little bit confusing because it pretends to be similar to `typeCheckPass` and `typeCheckFail` but has a completely different purpose and signature. Let’s just write this code explicitly inline in the corresponding test cases. Additionally, we can remove `typeCheckFail` calls from them because I think we only want the warning to appear once.